infra/build/developer-tools-light/README.md

```diff
@@ -0,0 +1,12 @@
+# Developer Tools Image Light
+
+This image is a light version of developer-tools with Terraform and Cloud SDK.
+
+## Building and Releasing
```

Comment:
Perhaps we should cut this section down and simply say to run the Makefile in the parent directory?

Reply:
done